src/store/UI/index.js

```diff
@@ -21,6 +21,7 @@ export function toggleSettings () {
 
 
 function UIReducer (state = initialState, action) {
+  console.log('UI', action);
```

Comment:
I recommend installing [this logging](https://github.com/evgenyrodionov/redux-logger) middleware to avoid having to drop console logs like leaves onto the autumn ground.

just a thought though.